### PR TITLE
Shanoir-issue#876 Update and use dcm2niix by default

### PR DESCRIPTION
**Pull request by jcomedouteau**
- Update dcm2niix version
- Set dcm2niix as default + tooltip

### CODE REVIEW

File: shanoir-ng-front/src/app/study-cards/study-card/study-card.component.ts [hunk 1]
```diff
@@ -144,7 +144,11 @@ export class StudyCardComponent extends EntityComponent<StudyCard> {
 
     private fetchNiftiConverters() {
         this.niftiConverterService.getAll()
-            .then(converters => this.niftiConverters = converters.map(converter => new IdName(converter.id, converter.name)));
+            .then(converters => {
+                this.niftiConverters = converters.map(converter => new IdName(converter.id, converter.name));
+                // pre-select dcm2niix
+                this.studyCard.niftiConverter = converters.filter(element => element.name === 'dcm2niix')[0];
```
Comment (Nunien) [inline shanoir-ng-front/src/app/study-cards/study-card/study-card.component.ts:150]: fails in edit mode, this.studyCard is undefined at this point. Anyway, you don't want to change the converter in edit mode, your dev is only for the create mode. So you must move that code to the initCreate() function :
* Just move one row up this.studyCard = new StudyCard(); (above this.fetchNiftiConverters();)
*  Add a .then to this.fetchNiftiConverters(); : 
    * this.fetchNiftiConverters().then(() => {   put your code here   });

Comment (Nunien) [inline shanoir-ng-front/src/app/study-cards/study-card/study-card.component.ts:150]: fails in edit mode, this.studyCard is undefined at this point. Anyway, you don't want to change the converter in edit mode, your dev is only for the create mode. So you must move that code to the initCreate() function :
* Just move one row up this.studyCard = new StudyCard(); (above this.fetchNiftiConverters();)
*  Add a .then to this.fetchNiftiConverters(); : 
    * this.fetchNiftiConverters().then(() => {   put your code here   });

**Review by Nunien — CHANGES_REQUESTED** [review]

**Comment by jcomedouteau** [comment]
@Nunien Corrected, well seen :) !

**Review by Nunien — APPROVED** [review]

**Comment by ebannier2018** [comment]
Hi, Can you keep me posted on this issue ? What is the current dcm2niix version ? When was it updated ? 
I still have an issue with some data imported on Friday April 9th. What was the dcm2niix version then ? see my email from April 14th + data attached. 
Thanks in advance
Elise

**Comment by jcomedouteau** [comment]
Hello Elise !

We have updated the dcm2niix version:
Chris Rorden's dcm2niiX version v1.0.20201102 

This converter is now used for new imports (since the 15th of April). For older data, it was an older version of dcm2niix which was in use.
For the moment we can't re-convert the existing data, but there is a pull request to resolve this. (It will allow to re-convert dataset by dataset with a specific converter)

Jean-Côme

**Comment by ebannier2018** [comment]
Thanks a lot Jean-Come. Too bad for the many imports from April 9th but it worked fine for today's import. 
Btw, there was a recent release of a 2021 version of dcm2niix : https://github.com/rordenlab/dcm2niix/releases
Maybe interesting to update for this one ?

**Comment by jcomedouteau** [comment]
Elise,
I had not seen it yet, I'll update it then !

Jean-Côme

**Pull request merged**